<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplicationProgressWizardPage</class>
 <widget class="QWizardPage" name="ReplicationProgressWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>499</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="createPublicationL">
       <property name="text">
        <string>Create publiction:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="createPublicationProgress">
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="showLogPB">
       <property name="text">
        <string>Show more info &gt;&gt;</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="logTE">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showLogPB</sender>
   <signal>toggled(bool)</signal>
   <receiver>ReplicationProgressWizardPage</receiver>
   <slot>showLogPB_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showLogPB_toggled(bool)</slot>
 </slots>
</ui>
